config.js

```diff
@@ -0,0 +1,26 @@
+var _ = require('lodash');
+var meConf = null;
+
+try {
+    meConf = require('./meConf');
+}
+catch(e) {
+    console.warn('(optional) meConf.json is missing.');
+}
+
+// Default Configuration
+var config = {
```

Comment:
please put default config under "prod.json" - we can use it as reference for me.json.